build/configs/rtl8720e/scripts/rlx8720e_img2_is.ld

```diff
@@ -278,7 +274,7 @@ SECTIONS
 		. = . + 4096;
 		_ebss = ABSOLUTE(.);
 
-	} > KM4_BD_PSRAM_NS_NTZ
+	} > KM4_BD_PSRAM_NS_TZ

```

Comment:
For Trustzone disabled, should this be KM4_BD_PSRAM_NS_NTZ ??

Reply:
Yes you are right, let me change it back!